.github/workflows/march-image-build-push.yml

```diff
@@ -0,0 +1,78 @@
+name: 'multi-arch image build'
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - 'main'
+      - 'release-*'
+    tags:
+      - 'v*'
+  schedule:
+    - cron: '0 0 * * *'
```

Comment:
Is this saying to build a new multi-arch image every day at midnight?

Reply:
Yes. I don't think there is a need for this; I may have carelessly copied it from something else I was working on (konveyor/builder) as a starting point.